.appveyor.yml

```diff
@@ -0,0 +1,117 @@
+build: false
+platform:
+  - x64
+#matrix:
+#  fast_finish: true # kills the build at the first failure
+clone_folder: C:\projects\dbal
+
+## Build matrix for lowest and highest possible targets
+environment:
+  dependencies: current
+  matrix:
+  - db: pgsql
+    php_ver_target: 7.0
+  - db: pgsql
+    php_ver_target: 7.1
+  - db: mssql
+    php_ver_target: 7.0
+  - db: mssql
+    php_ver_target: 7.1
+  - db: sqlsrv
+    php_ver_target: 7.0
+  - db: sqlsrv
+    php_ver_target: 7.1
+  - db: sqlite
+    php_ver_target: 7.0
+  - db: sqlite
+    php_ver_target: 7.1
+  - db: mysql
+    php_ver_target: 7.0
+  - db: mysql
+    php_ver_target: 7.1
+  - db: mysqli
+    php_ver_target: 7.0
+  - db: mysqli
+    php_ver_target: 7.1
+  - db: mariadb
+    php_ver_target: 7.0
+  - db: mariadb
+    php_ver_target: 7.1
+
+## Cache composer bits
+#cache:
+#    - '%LOCALAPPDATA%\Composer\files -> composer.lock'
+init:
+  - SET PATH=C:\Program Files\OpenSSL;c:\tools\php;%PATH%
+  - SET COMPOSER_NO_INTERACTION=1
+  - SET PHP=1
+  - SET ANSICON=121x90 (121x90)
+services:
+#  - mssql2014
+#  - mysql
+#  - postgresql94
+
+## Install PHP and composer, and run the appropriate composer command
+install:
+    - IF EXIST c:\tools\php (SET PHP=0)
+    - ps: appveyor-retry cinst --ignore-checksums -y php --version ((choco search php --exact --all-versions -r | select-string -pattern $Env:php_ver_target | Select-Object -first 1) -replace '[php|]','')
+    - cinst -y sqlite
+    - ps: If ($env:db -like "mariadb") {cinst -y mariadb}
+    - cd c:\tools\php
+    #get the MSSQL DLL's
+    - ps: >-
+        If ($env:php_ver_target -eq "7.0") {
+          If ($env:PHP -eq "1") {
+            If ($env:db -like "mssql") {
+              appveyor DownloadFile https://github.com/Microsoft/msphpsql/releases/download/4.1.5-Windows/7.0.zip
+              7z x 7.0.zip > $null
+              copy 7.0\x64\php_pdo_sqlsrv_7_nts.dll ext\php_pdo_sqlsrv_nts.dll
```

Comment:
I would copy the DLLs to `php_pdo_sqlsrv.dll` instead of `php_pdo_sqlsrv_nts.dll`. This makes the naming consistent with the other driver's DLLs and you can potentially make use of variable substitution to save `if...else` stuf